VultisigApp/VultisigApp/Views/Keysign/KeysignMessageConfirmView.swift

```diff
@@ -36,18 +36,10 @@ struct KeysignMessageConfirmView: View {
             }
             .foregroundColor(.neutral0)
             .onAppear {
-                if (viewModel.keysignPayload?.coin.chainType == .EVM) {
-                    isLoading = true
-                    Task{
-                        do {
-                            blowfishResponse = try await viewModel.blowfishEVMTransactionScan()
-                            isLoading = false
-                        } catch {
-                            print(error.localizedDescription)
-                            isLoading = false
-                        }
-                    }
-                } else {
+                isLoading = true
+                Task {
+                    try await viewModel.blowfishTransactionScan()
```

Comment:
if there is an exception somehow , this might cause the app the crash , because the exception has not been captured ,correct?

Reply:
Tks, I've fixed that one.